### ISSUE TITLE
[Feature Request]Upload组件的props.action不应该为必填

### ISSUE THREAD

**Issue by bowencool** [iview/iview#3850]
### What problem does this feature solve?
可以解决一个控制台报错：
```
[Vue warn]: Missing required prop: "action"
found in
    ---> <Upload>
```
### What does the proposed API look like?
我在`props.beforeUpload`中`return false`的时候，不应该报错`Missing required prop: "action"`

**Comment by David-Tsui** [comment]
`<Upload action="''" />`

**Comment by m430** [comment]
I will check it later and improve the `action` that should not be required.

**Comment by Vishal-Dhimmar** [comment]
<Upload action="' '"  /> 
I check it not working.

iview.js:33730 POST http://localhost:8080/'' 404 (Not Found)

**Comment by antonelepfl** [comment]
Agree. It should not be required. Any work around until it's implemented?
